src/libsystemd/sd-bus/sd-bus.c

```diff
@@ -232,6 +233,8 @@ static sd_bus* bus_free(sd_bus *b) {
 
 DEFINE_TRIVIAL_CLEANUP_FUNC(sd_bus*, bus_free);
 
+DEFINE_ORIGIN_ID_HELPERS(sd_bus);
```

Comment:
don't use the sd_ prefix for internal stuff, it's how we name exported stuff.

Reply:
This is a type definition, so it has to be sd_bus/sd_journal as that's how the types are called. I called the functions to match the type name, but I added a separate field instead.

Reply:
nope, we only call public types with sd_

internal types are not named like that. compare usec_t or union sockaddr_union or similar

Reply:
The objects acted upon are public types: sd_bus, sd_event and sd_journal